docs/how-to/build/eps.md

```diff
@@ -261,7 +261,7 @@ See more information on the OpenVINO Execution Provider [here](../reference/exec
 ```
 
 * `--use_openvino` builds the OpenVINO Execution Provider in ONNX Runtime.
-* `<hardware_option>`: Specifies the default hardware target for building OpenVINO Execution Provider. This can be overriden dynamically at runtime with another option (refer to [OpenVINO-ExecutionProvider.md](../reference/execution-providers/OpenVINO-ExecutionProvider.md) for more details on dynamic device selection). Below are the options for different Intel target devices.
+* `<hardware_option>`: Specifies the default hardware target for building OpenVINO Execution Provider. This can be overriden dynamically at runtime with another option (refer to [OpenVINO-ExecutionProvider](http://www.onnxruntime.ai/docs/reference/execution-providers/OpenVINO-ExecutionProvider.html#summary-of-options) for more details on dynamic device selection). Below are the options for different Intel target devices.
```

Comment:
please use relative links. ../../reference/execution-providers/OpenVINO-ExecutionProvider.html#summary-of-options

Reply:
Done and verified